src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,43 @@
 import React from 'react';
 import './MovieCard.scss';
+import PropTypes from 'prop-types';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ description, imdbUrl, title, imgUrl }) => (
+  <div className="card">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="/images/imdb-logo.jpeg" alt="imdb" />
```

Comment:
```suggestion
            <img src="./images/imdb-logo.jpeg" alt="imdb" />
```
без точки не подргужается картинка 
![image](https://user-images.githubusercontent.com/64437769/92019780-d0dc5c80-ed5f-11ea-9dba-88d8d5fe626f.png)